config/locales/en.yml

```diff
@@ -1,10 +1,59 @@
 en:
   helpers:
     fieldset:
+      arson:
+        arson: Arson?
+        damage_to_property: Damage to property?
+      communication:
+        can_read_and_write: Able to read and write?
+        hearing_speach_sight: Hearing / speech / sight?
+        interpreter_required: Interpreter required?
+      concealed_weapons:
+        conceals_weapons: Conceals weapons or other items?
+      harassments:
+        hostage_taker_stalker_harasser: Hostage taker / Stalker / Harasser / Intimidation / Bullying?
+      from_others:
+        rule_45: Rule 45?
+        csra: CSRA?
+        verbal_abuse: At risk of verbal abuse?
+        physical_abuse: At risk of physical abuse?
       move_information:
         has_destinations: Are there establishments the detainee must or must not be sent?
         reason: Reason for move
+      non_association_markers:
+        non_association_markers: Non-association markers?
+      security:
+        category_a: Category A?
+        current_e_risk: Current E risk?
+        escape_list: Escape list?
+        other_escape_risk_info: Is there other escape risk information?
+        restricted_status: Restricted status?
+      sex_offences:
+        sex_offence: Sex offence?
+      substance_misuse:
+        alcohol: Alcohol?
+        drugs: Drugs?
+      to_self:
+        open_acct: Open or recent ACCT?
+        suicide: Risk of suicide or self-harm?
+      violence:
+        violent: Violent?
     label:
+      arson:
+        arson:
+          unknown: Clear selection
```

Comment:
We need a better & consistent approach for managing the text on these yes/no/clears as opposed to just having it duplicated like this.